fetch/main.js

```diff
@@ -0,0 +1,9 @@
+const element = document.querySelector('.testing');
+
+fetch('https://pokeapi.co/api/v2/pokemon/1')
+  .then(response => {
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+    return response.json(element).then(data => console.log(data));
+  });
```

Comment:
The `.json` method doesn't take any arguments. It is better to chain the second `.then` to the first `.then` instead of directly onto the `.json` method.
```JAVASCRIPT
.then(response => {
  if (!response.ok) {
    throw new Error(`HTTP error! Status: ${response.status}`);
  }
  return response.json();
})
.then(data => console.log(data));
```